opencontrail/files/cassandra_backup

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -uef -o pipefail
+
+# Define vars
+BACKUP_DIR=/var/backups/cassandra
+DATA_DIR=/var/lib/cassandra/data
+NODETOOL=$(which nodetool)
+
+TODAY_DATE=$(date +%F)
+NODENAME=$(salt-call pillar.get linux:network:fqdn --out=newline_values_only)
```

Comment:
Use grains instead or just `hostname -f` and rely on correctly set hostname. Dependency on other formulas is not necessary.